### PR TITLE
Revert "Merge branch 'master' of github.com:vippsas/vipps-ecom-api"

### PR DESCRIPTION
**Pull request by rebeksburnett**
This reverts commit 82258e00b9c2b1e8eeb1e9920d0b664a87641ab9, reversing
changes made to 20eb93b4ee503962ed6ddd279ca3106bacd358d3.

### CODE REVIEW

**Comment by rebeksburnett** [comment]
I was working in my own fork, but it got pushed to the real repository by VSCode, and the branch was old, so it added a lot of weird stuff.

File: docs/shins/index.html [hunk 1]
```diff
@@ -1930,7 +1930,7 @@
                   </li>
                 
                   <li>
-                    <a href="#tocs_errorfromazure" class="toc-h2 toc-link" data-title="">ErrorFromAzure</a>
```
Comment (cloveras) [inline docs/shins/index.html:1933]: The `ErrorFromAzure` was a nice improvement, so I'd like to keep that.

File: docs/shins/index.html [hunk 2]
```diff
@@ -2185,7 +2185,7 @@ <h3 id="fetch-authorization-token-responses">Responses</h3>
 <tr>
 <td>default</td>
 <td>Default</td>
-<td>Possible error responses. Errors can be from both Vipps' own code (where we have full control) and from Microsoft Azure (where we rely on standard functionality). See the <a href="https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md">&quot;Common errors&quot; section of the FAQ</a>. See also: <a href="https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#url-validation">Responses</a>.</td>
+<td>Possible error responses. See <a href="https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#url-validation">Responses</a>.</td>
```
Comment (cloveras) [inline docs/shins/index.html:2188]: Same here: The `ErrorFromAzure` was a nice improvement, so I'd like to keep that.

File: docs/shins/index.html [hunk 3]
```diff
@@ -9503,23 +9503,22 @@ <h3 id="properties-41">Properties</h3>
 </tr>
 </tbody>
 </table>
-<h2 id="tocs_errorfromazure">ErrorFromAzure</h2>
-<p><a id="schemaerrorfromazure"></a>
-<a id="schema_ErrorFromAzure"></a>
-<a id="tocSerrorfromazure"></a>
-<a id="tocserrorfromazure"></a></p>
+<h2 id="tocs_errorobject">ErrorObject</h2>
```
Comment (cloveras) [inline docs/shins/index.html:9506]: The `ErrorFromAzure` was a nice improvement, so I'd like to keep that. We don't need the vague `ErrorObject`.

**Comment by rebeksburnett** [comment]
These changes are wanted. The revert was not the correct content.

**Pull request closed without merging**